boot/cypress/MCUBootApp/keys.c

```diff
@@ -174,3 +174,23 @@ struct bootutil_key bootutil_keys[1] = {
 };
 const int bootutil_key_cnt = 1;
 #endif /* !MCUBOOT_HW_KEY */
+
+unsigned char enc_priv_key[] = {
```

Comment:
Can you elaborate on where this private key came from? I'm curious why this is different than the equivalent value used on other platforms.

Reply:
those are same as enc-ec256-*.pem

removed duplication inside \cypress-folder